install-sonar-scanner-cli.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eou pipefail
+
+#See https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
+
+WGET=wget
+if [[ "$RUNNER_OS" == "Linux" &&  "$RUNNER_ARCH" == "X64" ]]; then
+     FLAVOR="linux-x64"
+elif [[ "$RUNNER_OS" == "Linux" &&  "$RUNNER_ARCH" == "ARM64" ]]; then
+     FLAVOR="linux-aarch64"
+elif [[ "$RUNNER_OS" == "Windows" &&  "$RUNNER_ARCH" == "X64" ]]; then
+     FLAVOR="windows-x64"
+     WGET="C:\\msys64\\usr\\bin\\wget.exe"
```

Comment:
This could be problematic on self-hosted Windows runners. WDYT @henryju?

Reply:
As pointed out by Evgeny, running on Windows was not supported until now, so this should not be a problem for all existing pipelines, which had to be running on Linux as they had to spawn a Docker container, and that is only supported by GitHub on Linux.

Reply:
I think I found this on an existing GitHub action (maybe not an official one). If people complain, we can improve later (For example try to locate using `where.exe`).